api/views/file.py

```diff
@@ -5,28 +5,25 @@
 from rest_framework.views import APIView
 
 from core.models import File
-from etl.tasks import import_file_task
+from etl.tasks import remove_file_task
 
-__all__ = ("FileUpdateView",)
+__all__ = ("FileDeleteView",)
 
 logger = logging.getLogger(__name__)
 
 
-class FileUpdateView(APIView):
+class FileDeleteView(APIView):
     """
-    A view to restart a file.
+    A view to remove a file.
     """
 
     permission_classes = [IsAuthenticated]
 
     def post(self, request):
-        # MVP: We assume this will always be one and only one failed file.
-        qs = (
-            File.objects.filter(account=request.data["id"])
-            .filter(import_status=File.FAILED)
-            .first()
+        qs = File.objects.filter(account=request.data["id"]).filter(
+            import_status=File.FAILED
         )
```

Comment:
``.first()`` should be added back here

Reply:
I couldn't tell from your slack, if you were still concerned about this, but I do think it's right to remove it based on the functionality we are going for.